account_tax_one_vat/data/account_tax_group.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" ?>
+<odoo noupdate="1">
+  <record id="vat_tax_group" model="account.tax.group">
```

Comment:
Hi. I think you should handle that point (know if it is a vat or not), by adding a is_vat on `account.tax.`(template)
It can occures that there are a vat group 5,5% and a vat group 10% and a vat group 20%. All that configuration are valid.

Reply:
Indeed, looks cleaner than relying on an xmlid